### PR TITLE
chore(ui): improve ui / ux around patch api calls

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes:

Fixes <issue-number>

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Modals/EntityNameModal/EntityNameModal.component.tsx [hunk 1]
```diff
@@ -28,14 +28,14 @@ const EntityNameModal: React.FC<EntityNameModalProps> = ({
...
-      await form.validateFields();
-      onSave(obj);
-    } catch (_) {
-      // Nothing here
-    }
+    setIsLoading(true);
+    await form.validateFields();
+    // Error must be handled by the parent component
+    await onSave(obj);
+    setIsLoading(false);
```
Comment (karanh37) [inline openmetadata-ui/src/main/resources/ui/src/components/Modals/EntityNameModal/EntityNameModal.component.tsx:38]: Shouldn't this loading be set to false in catch ?

File: openmetadata-ui/src/main/resources/ui/src/components/common/EntityImport/EntityImport.component.tsx [hunk 2]
```diff
@@ -71,10 +71,12 @@ export const EntityImport = ({
     try {
       const result = e.target?.result as string;
       if (result) {
+        setUploading(true);
         const response = await onImport(entityName, result);
 
         setCsvImportResult(response);
         setCsvFileResult(result);
+        setUploading(false);
```
Comment (karanh37) [inline openmetadata-ui/src/main/resources/ui/src/components/common/EntityImport/EntityImport.component.tsx:79]: preferrably move to finally

**Review by karanh37 — CHANGES_REQUESTED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/Modals/EntityNameModal/EntityNameModal.component.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/Modals/EntityNameModal/EntityNameModal.component.tsx:38]: As the error will be handled by the parent component, we don't need a catch block here!

File: openmetadata-ui/src/main/resources/ui/src/components/common/EntityImport/EntityImport.component.tsx (hunk 2 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/common/EntityImport/EntityImport.component.tsx]: push the changes

**Review by karanh37 — APPROVED** [review]

**Pull request merged**